masterlist.yaml

```diff
@@ -3732,13 +3735,36 @@ plugins:
 
 ###### Sim Settlements 2 ######
   - name: 'SS2.esm'
-    url: [ 'https://www.nexusmods.com/fallout4/mods/47976' ]
+    url:
+      - 'https://www.nexusmods.com/fallout4/mods/47976'
+      - 'https://bethesda.net/en/mods/fallout4/mod-detail/4183663'
+    after:
+      - 'Unofficial Fallout 4 Patch.esp'
```

Comment:
It sorts this way by default due to group rules:

```suggestion
```

Reply:
Ok, I was just trying the 'make it obvious' path...

Reply:
In case you forgot, you didn't actually commit this change.